.prettierignore

```diff
@@ -0,0 +1,12 @@
+# Make sure to keep contents aligned with all other interfaces!
+#
+# See .gitignore
+coverage/
```

Comment:
Will these ignore-rules now be taken into account when you open the Portal-only in VSCode?

So if you run "`npm test`" in the Portal-folder (at least once), the `coverage`-folder should be created... Subsequent runs of Prettier (in the Portal-folder) should not check these files then...

Reply:
Prettier won't run in the Portal-folder - eslint (with `npm run lint`) is the only thing being run on a sub-folder level (which already ignores the coverage folder).

The IDE might still "prettify" these files if you save them, but that isn't an issue IMHO because you aren't committing them anyway.

Reply:
Ah, yes.. makes sense. ESLint will be in charge of most things anyway indeed.

Prettier is just a "sub-contractor", so it won't have "the last say" anywhere...